src/blocks/logo.scss

```diff
@@ -0,0 +1,10 @@
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+
+  &__header {
```

Comment:
The BEM element '.logo__header' is defined but it is not clear if it is necessary. The styles applied to it are the same as its parent '.logo'. If '.logo__header' is not having any additional styles, it can be removed.